src/extension.ts

```diff
@@ -385,9 +385,23 @@ class ProbeRSDebugAdapterServerDescriptorFactory implements vscode.DebugAdapterD
             logToConsole(
                 `${ConsoleLogSources.console.toLowerCase()}: Launching new server ${JSON.stringify(
                     command,
-                )} ${JSON.stringify(args)} ${JSON.stringify(options)}`,
+                )} ${JSON.stringify(args)}}`,
```

Comment:
That wasn't really on purpose, but it annoyed me because it added a lot of lines to the terminal, so I removed it. I didn't really care about all the enviroment variables

But I will add it back.